config/locales/nl.yml

```diff
@@ -456,37 +456,37 @@ nl:
         voice_chat: 'spraak oproepen'
         training: training
         directory: lijst
-        self_injury: 'zelf verminking'
+        self_injury: 'zelfverminking'
         treatment: behandeling
-        care_planning: 'zorg planning'
+        care_planning: 'zorgplanning'
         depression: depressie
-        ocd: OCD
+        ocd: OCS
```

Comment:
Normally this is called OCD in healthcare. We use the English term. I prefer to keep OCD here.